### PR TITLE
Set network time to Redis

### PR DESCRIPTION
**Pull request by TronTram**
## Why
- Fix error: 
image

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'active_support/core_ext/integer/time'
+require 'sprockets/railtie'
```
Comment (jdenquin) [inline config/environments/production.rb:4]: we do not need `sprockets` for production

File: config/credentials/production.yml.enc
Comment (jdenquin) [inline config/credentials/production.yml.enc:1]: You can remove this file

File: Dockerfile
Comment (jdenquin) [inline Dockerfile:1]: We do not want to include the development libs for the image we publish, you can use the `Dockerfile.dev` for this!

File: .gitignore [hunk 2]
```diff
@@ -36,3 +36,5 @@ db/events_schema.rb
 LAGO_VERSION
 
 **/.DS_Store
+
+/config/credentials/production.key
```
Comment (jdenquin) [inline .gitignore:40]: You can remote it, we do not use those credentials

**Review by jdenquin — COMMENTED** [review]
Thanks for contributing, I put some comments here!

**Comment by TronTram** [comment]
> Thanks for contributing, I put some comments here!

thanks you so much.
 it's my mistake, it should merge to my fork repo.

**Pull request closed without merging**